### PR TITLE
Internal changes to optimal search schemes

### PR DESCRIPTION
**Pull request by cpockrandt**

### CODE REVIEW

File: include/seqan/index/find2_index_approx.h [hunk 1]
```diff
@@ -280,7 +280,7 @@ inline void _optimalSearchSchemeInit(std::array<OptimalSearch<nbrBlocks>, N> & s
         s.startPos = 0;
         for (uint8_t i = 0; i < s.pi.size(); ++i)
             if (s.pi[i] < s.pi[0])
-                s.startPos += s.blocklength[i] - ((i > 0) ? s.blocklength[i-1] : 0);
+                s.startPos += s.blocklength[i] - s.blocklength[i-1];
```
Comment (rrahn) [inline include/seqan/index/find2_index_approx.h]: there should be spaces between any operator and it's operands: `[i - 1]`.

**Comment by cpockrandt** [comment]
@rrahn Fixed!

**Comment by cpockrandt** [comment]
@rrahn Can you merge this? :)

**Comment by rrahn** [comment]
Lgtm

**Comment by rrahn** [comment]
Apparently I can’t merge from the phone if there are errors in the checks

**Pull request merged**